docs/DeveloperGuide.md

```diff
@@ -434,40 +438,39 @@ The following activity diagram summarizes what happens when a user executes a ne
-* prefers typing to mouse interactions
-* is reasonably comfortable using CLI (command-line interface) apps
+**Preferences/Skills**
+* Prefer desktop apps over other types
+* Can type fast
+* Prefers typing to mouse interactions
+* Is reasonably comfortable using CLI (command-line interface) apps
 
 **Value proposition**: streamlined text-based commands to manage contacts faster than a typical mouse/GUI driven app
 
 ### User stories
 
 Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unlikely to have) - `*`
 
-| Priority | As a …​                                | I want to …​                                | So that I can…​                                              |
-| ---- | -------------------------------------- |---------------------------------------------|--------------------------------------------------------------|
-| `* * *` | user                                   | add patient records                      | keep track of their medical condition in the hospital        |
-| `* * *` | user                                   | view existing patient records               | access information on existing patients                      |
-| `* * *` | user                                   | delete a patient record                     | remove outdated or irrelevant patient data                   |
-| `* *` | user                                   | edit patient records                        | ensure that all patient details are current and accurate     |
-| `* *` | user                                   | list patients based on their tags           | view patients based on category                              |
-| `* *` | user                                   | list all patients tied to a specific ward   | know which patients belong to which ward                     |
-| `* *` | user                                   | find patients via their name or NRIC        | quickly find information of specific patient                 |
-| `* *` | user                                   | access the user guide through the app easily | learn how to use the Nursing Address Book                    |
-| `*`  | user                                   | view patient list sorted by name            | look through the list of patients in a more organized manner |
+| Priority | As a …​ | I want to …​                                | So that I can…​                                                   |
+| ---- |---------|---------------------------------------------|-------------------------------------------------------------------|
+| `* * *` | nurse   | add patient records                      | keep track of their medical condition in the hospital             |
+| `* * *` | nurse   | view existing patient records               | access information on existing patients                           |
+| `* * *` | nurse   | delete a patient record                     | ensure that the system is up to date with the latest patient data |
```

Comment:
I think the older one would fit better...what do you think? Otherwise I think this fits the purpose as well.

Reply:
You mean the one with user instead of nurse?